testbed/src/testbed/models/lightning_uq_models/_card.py

```diff
@@ -26,10 +26,10 @@
 from torch.optim import Adam
 from tqdm import tqdm
 
-from testbed.models._preprocessors import Preprocessor
 from testbed.models.base_model import ProbabilisticModel
 from testbed.models.lightning_uq_models._data_module import GenericDataModule
 from testbed.models.lightning_uq_models._utils import _to_tensor
+from treeffuser.scaler import ScalerMixedTypes
```

Comment:
I am still a bit unsure about how to handle this situation where we depend on Treeffuser in a way that is beyond the Treeffuser class itself. I am not sure I like this but I don't like the solution that was there before of duplicating code. 
Just pointing it out.